soc/arm/nordic_nrf/nrf53/soc.c

```diff
@@ -168,6 +161,24 @@ bool z_arm_on_enter_cpu_idle(void)
 
 	return true;
 }
+
+bool z_arm_on_enter_cpu_idle(void)
+{
+	bool ok_to_sleep = nrf53_anomaly_160_check();
+
+#if (LOG_LEVEL >= LOG_LEVEL_DBG)
+	static bool suppress_message;
+
+	if (ok_to_sleep) {
+		suppress_message = false;
+	} else {
```

Comment:
Shouldn't this be `elseif (!suppress_message)` ?

Reply:
Fixed.